Ginger/GingerCoreNET/Run/Platforms/PlatformsInfo/WindowsPlatform.cs

```diff
@@ -364,6 +364,7 @@ public override List<eElementType> GetPlatformUIElementsType()
                 mElementsTypeList.Add(eElementType.Dialog);
                 mElementsTypeList.Add(eElementType.MenuItem);
                 mElementsTypeList.Add(eElementType.MenuBar);
+                mElementsTypeList.Add(eElementType.Image);
                 //mElementsTypeList.Add(eElementType.Browser);
```

Comment:
you need to also handle in GetDefaultElementOperation to set default operation for this element type

Reply:
Default operation for this type is IsExist